src/components/Filters/StateButton/StateButton.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Icon } from '../../Icon';
+import { IconTypes } from '../../Icon/Icon.enums';
+import { Text } from '../../typography/Text';
+import { FlexContainer } from '../../FlexContainer';
+import {
+  getBorderRadius,
+  getColor,
+  getDepth,
+  pxToRem,
+} from '../../../utils/helpers';
+import { StateButtonProps } from './StateButton.types';
+import { useStateButtonIcon } from '../hooks/useStateButton';
+
+const Popup = styled(FlexContainer)`
+  position: absolute;
+  visibility: hidden;
+  z-index: ${getDepth('tooltip')};
+  padding: ${pxToRem(8.5, 0, 8.5, 10)};
+  background: ${getColor('strawberry')};
+  border-radius: ${getBorderRadius} 0 0 ${getBorderRadius};
+  opacity: 0;
+  right: calc(100%);
+  top: 50%;
+  transform: translateY(-50%);
+`;
+
+const RemoveButton = styled(FlexContainer)`
```

Comment:
I would probably use `styled.button` directly as set flexbox properties. It looks odd when you use FlexContainer as button

Reply:
Yep, it is weird. It was my init version then I changed to FlexContainer. Will use `button`